custom_components/senseme/aiosenseme/__init__.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
```

Comment:
This file will be removed once https://github.com/mikelawrence/aiosenseme/pull/11 is merged